pkg/startupmigrations/migrations.go

```diff
@@ -479,7 +480,7 @@ func ExpectedDescriptorIDs(
 	defaultZoneConfig *zonepb.ZoneConfig,
 	defaultSystemZoneConfig *zonepb.ZoneConfig,
 ) (descpb.IDs, error) {
-	completedMigrations, err := getCompletedMigrations(ctx, db, codec)
+	completedMigrations, err := getCompletedMigrations(ctx, make(chan struct{}), db, codec)
```

Comment:
Can't passing the struct here lead to stalls during quiesce?
Ah I see the description says this is test only. I also no longer see this method on `master`. Ok, let's leave as is.

Reply:
Go land even shows me this method as grey and I don't see any use of it, so it is likely a leftover but I decided not to clutter the PR with unrelated change.